plugins/plugin-docker/che-plugin-docker-machine/src/test/java/org/eclipse/che/plugin/docker/machine/MachineProviderImplTest.java

```diff
@@ -318,6 +321,29 @@ public void shouldBeAbleToCreateContainerWithPrivilegeMode() throws Exception {
     }
 
     @Test
+    public void shouldBeAbleToCreateContainerWithCheUserId() throws Exception {
+        final String user = "test";
+        provider = new MachineProviderBuilder().setUser(user)
+                                               .build();
+        final boolean isDev = true;
+        CheServiceImpl service = createService();
+        service.setVolumes(null);
+        createInstanceFromRecipe(isDev, service);
+        ArgumentCaptor<CreateContainerParams> argumentCaptor = ArgumentCaptor.forClass(CreateContainerParams.class);
+        verify(dockerConnector).createContainer(argumentCaptor.capture());
+        verify(dockerConnector).startContainer(any(StartContainerParams.class));
+        String prepareCmd = "CHE_USER=`id -un`;"
+                + "sudo chown -R ${CHE_USER}:docker /home/user;"
+                + "sudo ln -s /home/user `eval echo \"~${CHE_USER}\"`";
+        final String[] command = {"/bin/sh", "-c", prepareCmd};
+        Exec exec = dockerConnector.createExec(CreateExecParams.create(CONTAINER_ID, command).withDetach(false));
```

Comment:
Here you call mocked methods. Instead of that it is needed to verify that they were called. Just like you did with **createContainer** and **startContainer**.